### PR TITLE
Prevent mouse from getting stuck

### PR DESCRIPTION
**Pull request by dginovker**
The mouse can get stuck a million ways. Let's put an end do that.

So far it's been working without any side effects (tested for ~5 mins)

### CODE REVIEW

File: .idea/runConfigurations.xml [hunk 1]
```diff
@@ -1,10 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (dginovker) [inline .idea/runConfigurations.xml:1]: Should this be committed?

No but it also shouldn't be in the project anyways so deal with it

File: .idea/shelf/Uncommitted_changes_before_Update_at_2_18_2022_2_24_AM__Default_Changelist_.xml [hunk 2]
```diff
@@ -1,4 +0,0 @@
-<changelist name="Uncommitted_changes_before_Update_at_2_18_2022_2_24_AM_[Default_Changelist]" date="1645169085463" recycled="true" deleted="true">
```
Comment (dginovker) [inline .idea/shelf/Uncommitted_changes_before_Update_at_2_18_2022_2_24_AM__Default_Changelist_.xml:1]: See above msg

**Review by myzsqn — APPROVED** [review]
Janky, but if it works, it works

File: .idea/runConfigurations.xml (hunk 1 above)
Comment (myzsqn) [reply .idea/runConfigurations.xml:1]: I wouldn't

**Pull request merged**